src/lighteval/models/base_model.py

```diff
@@ -345,6 +353,91 @@ def greedy_until_with_logits(
             override_bs=override_bs,
         )
 
+    def greedy_until_multi_turn(self, requests: list[GreedyUntilMultiTurnRequest], override_bs: Optional[int] = None) -> GenerateMultiTurnReturn:
```

Comment:
You'll also need to update the other model launchers, or at least add a function in the abstract model which indicates that it needs to be implemented